packages/adminrouter/docker/adminrouter-upstreams-ee.conf

```diff
@@ -1,17 +1,14 @@
 set $upstream_networking_api http://127.0.0.1:61430;
 
-proxy_ssl_trusted_certificate /run/dcos/pki/CA/certs/ca.crt;
+proxy_ssl_trusted_certificate /run/dcos/pki/CA/ca-bundle.crt;
 proxy_ssl_verify on;
 proxy_ssl_verify_depth 5;
 
-# For the sake of tests simplicity, we will not be using/testing leader.mesos
-# alias here. This should be done by integration tests.
-set $upstream_mesos http://127.0.0.2:5050;
-set $upstream_marathon http://127.0.0.1:8080;
+set $upstream_mesos http://leader.mesos:5050;
```

Comment:
Your comment made me thinking... 

In case of upstreams defined using DNS (like e.g. `/mesos`), we can't override DNS validity period with a value higher than the TTL set originally by mesos_dns. This gives us 60s upper limit.

As for the tests, the lower the better, with TTL=1 being the reasonable minimul.

As for the Lua code - this is a place where stuff starts to be interesting. Currently `CACHE_POLL_PERIOD = 25` seconds, we also want to be sure that `leader.mesos` will be re-resolved on every cache refresh, so max TTL of any DNS entry should be lower than 25s.

I think that we, on one hand, should not be using default TTL of the MesosDNS as it is too high, on the other hand, we should not query MesosDNS too often. Five seconds seems like a reasonable value. This shouldn't slow down our tests too much.

What do you think ?